public/src/sets.js

```diff
@@ -79,6 +79,7 @@ export default {
     "Arabian Nights": "ARN"
   },
   core: {
+    "Magic 2019": "M19",
```

Comment:
Regarding #122 it should be named `Core Set 2019`, see https://magic.wizards.com/en/products/core-2019